sdk/storage/Azure.Storage.Common/src/Shared/SasExtensions.cs

```diff
@@ -123,14 +123,14 @@ internal static string ToPermissionsString(this AccountSasServices services)
             {
                 sb.Append(Constants.Sas.AccountServices.Blob);
             }
-            if ((services & AccountSasServices.Queues) == AccountSasServices.Queues)
-            {
-                sb.Append(Constants.Sas.AccountServices.Queue);
-            }
             if ((services & AccountSasServices.Files) == AccountSasServices.Files)
             {
                 sb.Append(Constants.Sas.AccountServices.File);
             }
+            if ((services & AccountSasServices.Queues) == AccountSasServices.Queues)
```

Comment:
Moved this down for consistency with the ordering the Portal uses when constructing the Sas signature.

Reply:
Mind adding a comment in the source explaining why the order's important?

Reply:
Added